psi4/driver/procrouting/dft_funcs/dict_hyb_funcs.py

```diff
@@ -457,6 +457,143 @@ def get_pw6b95_tweaks():
     "citation": '    Pernal et. al., Phys. Rev. Lett., 103, 263201, 2009\n',
 })
 
+funcs.append({
+    "name": "M05",
+    "x_functionals": {
+        "HYB_MGGA_X_M05": {}
+    },
+    "x_hf": {
+        "alpha": 0.28
```

Comment:
You might want to use the `"HYB_MGGA_X_M05" : {"use_libxc": True}` instead of "hard-coding" alpha, such as in: 
https://github.com/psi4/psi4/blob/6e1c306b38828bdeea54e8718c6270e8b3a449f2/psi4/driver/procrouting/dft_funcs/dict_hyb_funcs.py#L214-L227

Reply:
Worked for all but M11. Thanks!